pkgs/development/python-modules/dunamai/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, poetry
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, importlib-metadata
+, packaging
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "dunamai";
+  version = "1.7.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "mtkennerly";
+    repo = "dunamai";
+    rev = "v${version}";
+    sha256 = "sha256-ocPkklKJTIT9YizQ4HwDv/r+9G+/XPxMYAgf9DuzDto=";
+  };
+
+  nativeBuildInputs = [
+    poetry
```

Comment:
Perhaps https://github.com/mtkennerly/dunamai/pull/35 gets merged quickly then we could move to `poetry-core`.